sdk/eventhub/azure-eventhub/out/azure/eventhub/aio/_consumer_client_async.pyi

```diff
@@ -0,0 +1,21 @@
+class EventHubConsumerClient(ClientBaseAsync):
+
+    #...
+
+    async def receive_batch(
```

Comment:
Is this the name the other SDKs are using?
I'm wondering if it maybe should be something like `receive_in_batches` or `receive_by_batch`.
Maybe it's just that I can't mentally separate the previous behaviour... but I feel like `receive_batch` translates to "give me a batch".....
@KieranBrantnerMagee  - am I over-thinking this?